tests/common/mock_command.py

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env python3
+import os
+import sys
+
+if len(sys.argv) != 4:
```

Comment:
This check would imply <4 is also a good number. Maybe change the condition to `len(sys.argv) < 4:` instead?

Reply:
it requires exactly 4 (stdout, return value & stderr + the command itself)

Reply:
btw since the script is not meant to be used manually (at least not frequently) i did not put any effort on setting default values for the parameters

Reply:
Ohh ok makes sense